cni: panic when testing standalone with cnitool

cnitool uses a very short ContainerID. This causes a panic in the plugin. This is a minor nit and should not matter in production. But standalone validation with cnitool fails.

https://github.com/containernetworking/cni/blob/master/cnitool/cni.go#L57
```
	rt := &libcni.RuntimeConf{
		ContainerID: "cni",
		NetNS:       netns,
		IfName:      "eth0",
	}
```

A bounds check is needed here
https://github.com/Azure/azure-container-networking/blob/master/cni/network/network.go#L89
```
// GetEndpointID returns a unique endpoint ID based on the CNI args.
func (plugin *netPlugin) getEndpointID(args *cniSkel.CmdArgs) string {
	return args.ContainerID[:8] + "-" + args.IfName
}
```

```
panic: runtime error: slice bounds out of range

goroutine 1 [running]:
github.com/Azure/azure-container-networking/cni/network.(*netPlugin).Add(0xc4200d0560, 0xc420060070, 0x3, 0x0)
        /home/tester/go/src/github.com/Azure/azure-container-networking/cni/network/network.go:142 +0x1982
github.com/Azure/azure-container-networking/cni.(PluginApi).Add-fm(0xc420060070, 0xc4200696d0, 0x5)
        /home/tester/go/src/github.com/Azure/azure-container-networking/cni/plugin.go:94 +0x39
github.com/containernetworking/cni/pkg/skel.(*dispatcher).checkVersionAndCall(0xc42006ee80, 0xc420060070, 0x7da060, 0xc42006d7d0, 0xc42004deb0, 0x0, 0x0)
        /home/tester/go/src/github.com/containernetworking/cni/pkg/skel/skel.go:162 +0x19f
github.com/containernetworking/cni/pkg/skel.(*dispatcher).pluginMain(0xc42006ee80, 0xc42004deb0, 0xc42004de98, 0x7da060, 0xc42006d7d0, 0x0)
        /home/tester/go/src/github.com/containernetworking/cni/pkg/skel/skel.go:173 +0x2f8
github.com/containernetworking/cni/pkg/skel.PluginMainWithError(0xc42004deb0, 0xc42004de98, 0x7da060, 0xc42006d7d0, 0xc42006d7d0)
        /home/tester/go/src/github.com/containernetworking/cni/pkg/skel/skel.go:210 +0xed
github.com/Azure/azure-container-networking/cni.(*Plugin).Execute(0xc420080030, 0x7d9f20, 0xc4200d0560, 0x0, 0xc4200642d0)
        /home/tester/go/src/github.com/Azure/azure-container-networking/cni/plugin.go:94 +0xd6
main.main()
        /home/tester/go/src/github.com/Azure/azure-container-networking/cni/network/plugin/main.go:35 +0x328
netplugin failed but error parsing its diagnostic message "": unexpected end of JSON input
```

--- a/cni/network/network.go
+++ b/cni/network/network.go
@@ -86,7 +86,13 @@ func (plugin *netPlugin) Stop() {
 
 // GetEndpointID returns a unique endpoint ID based on the CNI args.
 func (plugin *netPlugin) getEndpointID(args *cniSkel.CmdArgs) string {
-	return args.ContainerID[:8] + "-" + args.IfName
+	var containerID string
+	if len(args.ContainerID) >= 8 {
+		containerID = args.ContainerID[:8] + "-" + args.IfName
+	} else {
+		containerID = args.ContainerID + "-" + args.IfName
+	}
+	return containerID
 }
 
 // FindMasterInterface returns the name of the master interface.

